torch/csrc/jit/script/module.cpp

```diff
@@ -120,6 +120,20 @@ void Module::to_impl(
         non_blocking);
     variable.set_data(new_data);
   }
+  // Then convert every tensor attributes (buffers).
+  for (auto& parameter : get_attributes()) {
+    if (parameter.type()->isSubtypeOf(TensorType::get())) {
+      // Need to access the `at::Tensor` as a `Variable` here.
+      autograd::Variable variable = parameter.value().toTensor();
```

Comment:
This should be moved out to a function so it's not just a copy-paste of the above code for parameters